src/views/PVChargeConfig.vue

```diff
@@ -65,6 +65,13 @@
             <template #help> Obere Grenze des Regelbereichs. </template>
           </openwb-base-number-input>
           <hr />
+          <openwb-base-alert subtype="warning" v-if="
+            ($store.state.mqtt['openWB/general/chargemode_config/pv_charging/switch_on_threshold']) - 
+            ($store.state.mqtt['openWB/general/chargemode_config/pv_charging/switch_off_threshold']) < 1400">
+            Der einphasige Mindestladestrom von 6A ergibt eine Ladeleistung von etwa 1,4kW. Ein- und 
+            Ausschaltschwelle sollten daher mindestens 1,4kW auseinander liegen, um ständiges 
+            Ein- und Ausschalten zu vermeiden.
+          </openwb-base-alert>
```

Comment:
Das Problem ist, dass die Warnmeldung das komplette Layout verschiebt. Dadurch können ggf. Events (Maustaste gedrückt/losgelssen) nicht mehr korrekt zugeordnet werden. Ich würde den Hinweis daher immer anzeigen, wenn er zwingend notwendig ist. (Ergänzung hierzu im nächsten Kommentar.)
```suggestion
          <openwb-base-alert subtype="warning">
            Der einphasige Mindestladestrom von 6A ergibt eine Ladeleistung von etwa 1,4kW. Ein- und 
            Abschaltschwelle sollten daher mindestens 1,4kW auseinander liegen, um ständiges 
            Ein- und Ausschalten zu vermeiden.
          </openwb-base-alert>
```

Anmerkung: Bitte einheitliche Begriffe verwenden. Weiter unten heißt es "Abschaltschwelle", nicht "Ausschaltschwelle".